docs/data/stashcache/install-origin.md

```diff
@@ -11,87 +11,89 @@ to export its data to the StashCache data federation.
-  A RHEL 7-based operating system is strongly recommended.
 * Obtain root access to the host
 * Prepare [the required Yum repositories](/common/yum.md)
 * Install [CA certificates](/common/ca.md)
 
-Installing the StashCache origin
---------------------------------
+Installing the origin
+---------------------
 
-The StashCache daemon consists of an XRootD server and an HTCondor-based service for collecting and reporting
-monitoring data about the origin. To simplify installation, OSG provides convenience RPMs that install all required
+The origin service consists of an XRootD daemons and its dependencies for the authentication infrastructure.
+To simplify installation, OSG provides convenience RPMs that install all required
 software with a single command:
 
 ```console
-root@host # yum install stashcache-origin-server
+root@host # yum install --enablerepo=osg-development stash-origin
 ```
 
+!!! note
+    This document covers a completely overhauled stash origin service that depends on XRootD 4.9.0; while this
+    has not yet been released into the OSG production repository, we strongly recommend using this version.
+
 For this installation guide, we assume that the data to be exported to the federation is mounted at `/stash`
 and owned by the `xrootd:xrootd` user.
 
 Configuring the Origin Server
 -----------------------------
 
-The `stashcache-daemon` package provides a default configuration file,
-`/etc/xrootd/xrootd-stashcache-origin-server.cfg`, which must be customized for your origin.
+The `stash-origin` package provides a default configuration files in
+`/etc/xrootd/xrootd-stash-origin.cfg` and `/etc/xrootd/config.d`.  Administrators may provide additional configuration
+by placing files in `/etc/xrootd/config.d` of the form `/etc/xrootd/config.d/1*.cfg` (for directives that need to
+be processed BEFORE the OSG configuration) or `/etc/xrootd/config.d/9*.cfg` (for directives that are processed
+AFTER the OSG configuration).
+
+All _mandatory_ configuration that the admin must make is in `/etc/xrootd/config.d/10-origin-site-local.cfg`; EVERY
+directive there must be configured.
 
-The most common lines to customize are:
+The three mandatory directives that must be customized are:
 
-* `oss.localroot /stash`: Change the `localroot` to the location where your data is mounted on
-  the origin server; default is `/stash`.
-* `all.export /<YOUR VO>`: A sub-directory within the `localroot` directory that will be exported.
-  If multiple directories must be exported, you may specify `all.export` multiple times.
+* `origindir`: the mounted filesystem path to export.  This document refers to this as `/stash`.
+* `originexport`: the file within the `origindir` to export.  This MUST be in your supported VO's registered namespace.
+* `sitename`: the resource name registered with OSG.
 
 For example, if the HCC VO would like to set up an origin server exporting from the mountpoint `/mnt/bigdata`,
```

Comment:
If we say that we're going to refer to the mount point as `/stash` throughout the document, then we should do so -- i.e. let's replace `/mnt/bigdata` with `/stash` everywhere.

Reply:
Done!